index.html

```diff
@@ -11,58 +11,69 @@
 </head>
 </html>
 <body>
-    <div class="toast-container position-fixed bottom-0 end-0 p-3">
-        <div id="game-message" class="toast align-items-center text-bg-primary border-0" role="alert" aria-live="assertive" aria-atomic="true">
-          <div class="d-flex">
+    <div id="gamearea">
+        <div id="gamearea-popup" class="toast" role="alert">
             <div class="toast-body">
-                <a id="current-message"></a>
+                Welcome to the Craps Table!
+                <div class="mt-2 pt-2 border-top">
+                    <button type="button" class="btn btn-primary">Start Game</button>
```

Comment:
I noticed that when I push this button it doesn't do anything 🤔

Reply:
I've enabled the 'Start Game' button and ensured the screen view is normal after